app/styles/module/article/_portable-infobox.scss

```diff
@@ -85,6 +85,67 @@ $infobox-box-shadow: 0 6px 10px 0 rgba($wds-color-black, .1);
 
 //////////// General classes and rules used across infobox ////////////
 
+.pi-section-navigation {
+  $overflow-color: rgba($wds-color-black, 0.25);
+
+  @extend .wds-tabs;
```

Comment:
would that mean that styles for wds-tabs will end up twice in CSS?
could you add class to pi-section-navigation so it looks/behaves like a tab?

There are separate templates for infoboxes for mobile in app.